src/components/main-view/main-view.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import MovieCard from '../movie-card/movie-card';
+import MovieView  from '../movie-view/movie-view';
+
+const MainView = () => {
+
+    const [ movies, setMovies ] = useState([
```

Comment:
Better you declare this constant as an initial value some where under utils folder and the reference it. to keep you're components small and readable

/utils/dummyData.ts  `for example` and the reference it.